### ISSUE TITLE
Constructor inheritance and default values

### ISSUE THREAD

**Issue by andrievskya** [sabomichal/immutable-xjc#29]
Default no-args constructor for class that have ancestors calls constructor with arguments and passes nulls.

For example if we have two classes Foo and Bar, that extends it and have default value defined for an element of Foo:
```java
public class Foo {
    private final String param1;

    public Foo(){
        param1="Default1";
    }
    public Foo(String param1){
        this.param1 = param1;
    }
    //..
}
public final class Bar extends Foo {
    private final String param2;

    public Bar(){
        super(null);// Foo have default value for param1, but it will not be set
        this.param2="Default2";
    }
    //..
}
```
If we create an instance of Bar class using no-args constructor (for example, if we parse some xml without param1 specified), we will have an object with param1==null, and not "Default1"

**Comment by andrievskya** [comment]
I would suggest either removing the call to superclass constructor from generateStandardConstructor(...) method completely or adding a condition

**Comment by sabomichal** [comment]
Fixed in PR #32
